portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -16,16 +16,26 @@
   /* The time and date when the comment was submitted. */
   private String timeSubmitted;
 
+  /* Millisecond time representation when the comment was submitted. */
+  private long sysTime;
```

Comment:
This is nitpicky, but the `sysTime` name doesn't really tell the reader what the variable does. The comment is great, but I would maybe go one step further and make the name more obvious. How do you feel about changing this to `timeSubmitted` and changing the `String` variable to `formattedTimeSubmitted`?